src/mixxx.cpp

```diff
@@ -321,6 +321,7 @@ void MixxxMainWindow::initialize(QApplication* pApp, const CmdlineArgs& args) {
             &PlayerManager::noVinylControlInputConfigured,
             this,
             &MixxxMainWindow::slotNoVinylControlInputConfigured);
+    PlayerInfo::create();
```

Comment:
Is the place here reasonable? 
The decks and samplers are added later.

Reply:
I think that's okay. I needs to be created after enginemaster and playermanager. For updating track info it just loops over the deck indices. The number of decks is determined by the num_decks CO created by playermanger. That CO is updated each time you add a deck.